File: Java/Categoria Iniciante/URI 1047 - Tempo de jogo com minutos.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner (System.in);
		
		int h1, m1, h2, m2, hm1, hm2, H24, resultado;
		
		h1 = sc.nextInt();
		m1 = sc.nextInt();
		h2 = sc.nextInt();
		m2 = sc.nextInt();
		hm1 = (h1 * 60) + m1;
		hm2 = (h2*60) + m2;
		H24 = 24*60;
		resultado = 0;
		
		if( hm2 > hm1) {
			resultado = hm2 - hm1;
			System.out.printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", (resultado / 60), (resultado % 60));
		}	
		
		else if (hm1 > hm2) {
			resultado = (H24 - hm1) + hm2;
			System.out.printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n" , (resultado / 60), (resultado % 60));
		}
		else {
			System.out.printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n" , 24 , 0);
		}
		
		
		sc.close();
		
	}
}

File: C#/Categoria Iniciante/URI 1047 - Tempo de Jogo com minutos.cs
string [] entradas = Console.ReadLine().Split(' ');
int hi = int.Parse(entradas[0]);
int mi = int.Parse(entradas[1]);
int hf = int.Parse(entradas[2]);
int mf = int.Parse(entradas[3]);
int instanteInicial = (hi*60) + mi;
int instanteFinal = (hf*60) + mf;
int duracao;

if (instanteInicial < instanteFinal)
{
    duracao = instanteFinal - instanteInicial;
}

else
{
    duracao = (24*60 - instanteInicial) + instanteFinal;
}

int duracaoHoras = duracao/60;
int duracaoMinutos = duracao % 60;

Console.WriteLine("O JOGO DUROU " + duracaoHoras + " HORA(S) E " + duracaoMinutos + " MINUTO(S)");